recipes/mender-client/debian-master/rules

```diff
@@ -7,10 +7,10 @@ override_dh_auto_build:
 	make CGO_CFLAGS="${CGO_CFLAGS}" CGO_LDFLAGS="${CGO_LDFLAGS}" CC="${CC}" \
 	GOOS=linux CGO_ENABLED=1 build
 
+# All install targets except install-systemd
+install_targets = $(shell egrep ^install\-.*: Makefile | grep -v install-systemd | cut -d: -f1)
```

Comment:
seems hacky -- thats why I like it. what if we decide to add another target in the Makefile called install-systemd-redux? I am not super sure about parsing Makefile in here. but you are the expert.

Reply:
I agree that is hacky.

However the current approach has already proven difficult to maintain (we keep forgetting to add targets). Weird targets in the future could break this, but that is not worse than today's situation IMO.

In the other hand, your future example would do no harm, as it will install the `-redux` and every other target, so in effect it will install all (which is what I think we want overall).

Reply:
> In the other hand, your future example would do no harm, as it will install the -redux and every other target, so in effect it will install all (which is what I think we want overall).

This kind of happens now, doing `make ... install-inventory-scripts install-inventory-local-scripts install-inventory-network-scripts ...`, which is similar to your `-redux` example.

Reply:
ok then.